parser.go

```diff
@@ -64,12 +65,48 @@ func ParseUnverifiedClaimsFromJwtStringsRaw(jwtStrings []string) (validClaim *Cl
 		// We use the most recent token
 		if err != nil {
 			errList = append(errList, err)
-		} else {
-			if validClaim == nil || (claims.IssuedAt > validClaim.IssuedAt) {
+			continue
+		}
+
+		if validClaim == nil {
+			validClaim = claims
+			raw = jwtString
+			continue
+		}
+
+		reg, ok := validClaim.Claims.(jwt.RegisteredClaims)
+		if ok {
+			if reg.IssuedAt.Before(reg.IssuedAt.Time) {
 				validClaim = claims
 				raw = jwtString
 			}
+			continue
 		}
+
+		if validClaim.StandardClaims.IssuedAt > claims.StandardClaims.IssuedAt {
+			validClaim = claims
+			raw = jwtString
+		}
+
 	}
 	return
 }
+
+func ParseUnverifiedSpireJWT(ctx context.Context, token string) (validClaim *Claims, err error) {
+
+	svid, err := jwtsvid.ParseInsecure(token, []string{})
+
+	if err != nil {
+		return nil, err
+	}
+	_, claims, err := SVIDToGoJWT(svid)
+	if err != nil {
+		return nil, err
+	}
+
+	c := &Claims{
+		// TODO: Populate Claims with SVID information
```

Comment:
`FIXME:` The atlas_claims fields need to be populated with necessary information. Alternatively, OPA can be modified to read the embedded complete jwt in Claims.Claims field.